src/Flicking.js

```diff
@@ -384,6 +390,13 @@ export default class Flicking extends Mixin(Component).with(eventHandler) {
 		);
 	}
 
+	_clonePanel(v) {
+		const clone = v.cloneNode(true);
+
+		clone.classList.add(`${this.options.prefix}-clone`);
```

Comment:
I think `utils.classList` is more preferred for browser compatibility.

Reply:
Thank you!